frontend/src/features/env/SwitchEnvFeedbackModal.tsx

```diff
@@ -147,82 +146,115 @@ export const SwitchEnvFeedbackModal = ({
         ) : (
           <chakra.form noValidate onSubmit={handleFormSubmit}>
             <ModalHeader pr="48px">
-              Something not right on the new FormSG?
+              {user
+                ? 'Something not right on the new FormSG?'
+                : "Can't submit this form?"}
             </ModalHeader>
             <ModalBody mt="0" pt="0">
               <Stack spacing="1rem">
                 <FormControl>
                   <Input type="hidden" {...register('url')} value={url} />
                 </FormControl>
-                <FormControl
-                  isRequired
-                  isInvalid={!isEmpty(errors) || !!othersInputError}
-                >
-                  <FormLabel>
-                    Why are you switching to the previous FormSG?
-                  </FormLabel>
-                  <Radio.RadioGroup>
-                    {COMMON_RADIO_OPTIONS.map((option) => (
-                      <Radio
-                        {...register('switchReason', {
-                          required: {
-                            value: true,
-                            message: 'This field is required',
-                          },
-                          deps: [FEEDBACK_OTHERS_INPUT_NAME],
-                        })}
-                        value={'I’m not used to the new FormSG'}
-                        key={option}
-                        tabIndex={1}
-                      >
-                        {option}
-                      </Radio>
-                    ))}
-                    {radioOptions.map((option) => (
-                      <Radio
+                {user && isAdminView ? (
+                  <FormControl
+                    isRequired={isAdminView}
+                    isInvalid={
+                      isAdminView && (!isEmpty(errors) || !!othersInputError)
+                    }
```

Comment:
/nit `isAdminView` check not needed, since it's been tested for truthiness for the whole block already

```suggestion
                    isRequired={true}
                    isInvalid={
                      (!isEmpty(errors) || !!othersInputError)
                    }
```

Reply:
[9859f6d](https://github.com/opengovsg/FormSG/pull/5339/commits/9859f6ddfd48411620160e2caf5cfd34cd391c14)